docs/source/python_api/mlflow.pipelines.rst

```diff
@@ -18,3 +18,12 @@ Regression Pipeline
 .. autoclass:: mlflow.pipelines.regression.v1.pipeline.RegressionPipeline()
     :members:
     :undoc-members:
+
+Classification Pipeline
+~~~~~~~~~~~~~~~~~~~~~~~
+
+.. automodule:: mlflow.pipelines.classification.v1.pipeline
+
+.. autoclass:: mlflow.pipelines.classification.v1.pipeline.ClassificationPipeline()
```

Comment:
Classification pipeline class members don't seem to render in the generated docs (https://output.circle-artifacts.com/output/job/eb41f865-1ada-416f-af1d-d7c39639852a/artifacts/0/docs/build/html/python_api/mlflow.pipelines.html#mlflow.pipelines.classification.v1.pipeline.ClassificationPipeline)

<img width="995" alt="Screen Shot 2022-10-21 at 12 01 43 PM" src="https://user-images.githubusercontent.com/39497902/197270348-dbe0732d-ff9e-4293-a504-54814b074f96.png">

Reply:
Ah nice catch, I didn't override methods that just deferred to their super class. Will update.